docs/README-test-execution.md

```diff
@@ -0,0 +1,57 @@
+# Executing the tests provided for this gem
+
+## RSpec tests
+
+The test files located in the `spec/` directory use [RSpec](http://rspec.info/) as their test framework. These tests are generally standalone and do not generally require any actual Cisco hardware or virtual machines to test against.
+
+### Running a single RSpec test
+
+You can execute a single spec file by name:
+
+```bash
+rspec spec/environment_spec.rb
+```
+
+### Running all RSpec tests
+
+
+```bash
+rake spec
+```
+
+## Minitest tests
```

Comment:
Meh. `minitest` is the name of the test module that we're building on. I'd rather not change this.